corehq/ex-submodules/casexml/apps/phone/utils.py

```diff
@@ -38,8 +41,20 @@ def delete_sync_logs(before_date, limit=1000, num_tries=10):
     raise CouldNotPruneSyncLogs()
 
 
-def delete_sql_synclogs(before_date):
-    SyncLogSQL.objects.filter(date__lt=before_date).delete()
+def prune_sql_synclogs():
+    """
+    Drops all partition tables containing data that's older than 63 days (9 weeks)
```

Comment:
Instead of this can you the comment & `9 * 7` later, can you make a constant `SYNCLOG_RETENTION_WEEKS` ?

Also I thought we were planning to  keep them for ~90 days from the earlier PR? did we change that back to ~60?

Reply:
I will make it 90 days and make it a constant